app/services/schema_service.rb

```diff
@@ -0,0 +1,8 @@
+class SchemaService
+  def self.all_schemas
+    format_regex = Regexp.new(".+\/formats\/([a-z_]+)\/.+")
```

Comment:
so here we are using the file path to get the type of object the schema is for? Is there any other way to get the type, e.g. in the schema itself with a `title` key for instance? Just wondering if there's a risk depending on a file path set in another gem which could easily change...

Reply:
After chat with @pezholio There is no codified place for the name/type except in folder name